stdlib/3/signal.pyi

```diff
@@ -46,6 +48,45 @@ SIGWINCH = ...  # type: int
 SIGXCPU = ...  # type: int
 SIGXFSZ = ...  # type: int
 
+if sys.version_info >= (3, 5):
+    class Signals(IntEnum):
+        # duplicated exactly from signals above
```

Comment:
Hm, I thought of another thing. The type of the global variables (`SIGHUP` etc.) should also be Signals instead of int for Python >= 3.5. Someone could write things like `if isinstance(x, signal.Signals):` where x is initialized from e.g. `signal.SIGHUP`.